lib/rust/ensogl/core/src/display/scene/layer.rs

```diff
@@ -707,9 +734,29 @@ impl LayerModel {
         }
     }
 
-    /// Create a new sublayer to this layer, with the same camera.
+    /// Create a new sublayer to this layer. It will inherit this layer's camera.
     pub fn create_sublayer(&self, name: impl Into<String>) -> Layer {
-        let layer = Layer::new_with_cam(name, &self.camera.borrow());
+        let layer = Layer::new_with_flags(name, true, true);
```

Comment:
Such an API, where we are passing `true, true` or any combination is very hard to read. Please create newtypes / enums for expressing that instead.